testsuite/cluster-test/src/main.rs

```diff
@@ -148,6 +148,16 @@ pub fn main() {
             ));
             return;
         }
+    } else if args.health_check && args.swarm {
+        let mut util = BasicSwarmUtil::setup(&args);
+        util.logs = Some(DebugPortLogThread::spawn_new(&util.cluster).0);
```

Comment:
Is `BasicSwarmUtil` every reads `logs`?
It feels like you just set up logs here, put it into BasicSwarmUtil and then read here?

Reply:
I put logs and health_check_runner as member of BasicSwarmUtil. HealthCheckRunner will read logs.

Reply:
What I am trying to say here, is that you don't seem to need `log` as a field on BasicSwarmUtil?
You can just define `let logs = DebugPortLogThread::spawn_new` here

Reply:
Got it.